### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by nikunjvf**
Initial submit custom connector

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@nikunjvf I can see that you have made a submission in the ISV Studio to get your connector certified, but in the open-source repo, you have placed in the custom-connecors folder. 

Please note that the custom connectors placed in the custom-connectors folder never get published and certified. If you want to get the connector certified, you need to move the connector to "certified-connectors" or "verified-connectors" folder. 

If you still want your connector to be open sourced as a custom connector, please request us in the ISV Studio to abandon the submission you have made.

**Comment by Amjed-Ayoub** [comment]
Hi @nikunjvf I hope you are doing well.
If this is a verified connector, please follow the instructions below.
1.	Register your connector by filling out our [registration form](https://aka.ms/ConnectorRegistration). We will review it within 2 business days.
2.	Ensure that your connector complies with Microsoft requirements, as specified in [Meet submission requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-2-meet-submission-requirements).
3.	Prepare a set of files called connector artifacts to Microsoft by following the instructions in Prepare the connector artifacts..
4.	Ensure that your connector [contains valid metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata).
5.	Submit your connector by following the instructions in Submit your connector to Microsoft for certification.
a.	Open source the connector files. Our Certification Engineers will review within 1-2 weeks, and you will be notified in GitHub.
b.	Upload from local files to ISV Studio (If you've been granted an exception from open sourcing). We will review your submission within 1-2 weeks.
c.	After the pull request is merged, you may [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio), please allow us 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio.
6.	[Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to test your connector after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
7.	If your connector passes all tests, the deployment process begins, and it typically takes up to 3-4 weeks to deploy your connector to all our regions.
For detailed steps and instructions, please see our [Certification process](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission) page.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @nikunjvf i just noticed that your connector is registered. Please fix this pull request, you need to move the connector to "certified-connectors" or "verified-connectors" folder. thank you very much.

**Comment by nikunjvf** [comment]
> Hi @nikunjvf i just noticed that your connector is registered. Please fix this pull request, you need to move the connector to "certified-connectors" or "verified-connectors" folder. thank you very much.

Moved to proper certified connector folder. Thanks

File: certified-connectors/Vineforce/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/services/app/ExternalTask/CreateExternalTask": {
+      "post": {
+        "responses": {
+          "default": {
```
Comment (sriyen-msft) [inline certified-connectors/Vineforce/apiDefinition.swagger.json:24]: @nikunjvf Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: certified-connectors/Vineforce/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/services/app/ExternalTask/PushNotificationFromExternal": {
+      "post": {
+        "responses": {
+          "default": {
```
Comment (sriyen-msft) [inline certified-connectors/Vineforce/apiDefinition.swagger.json:136]: @nikunjvf Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: certified-connectors/Vineforce/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+            "description": "default"
+          }
+        },
+        "summary": "Alert",
```
Comment (sriyen-msft) [inline certified-connectors/Vineforce/apiDefinition.swagger.json:140]: @nikunjvf Please provide a sentence for summary values

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Vineforce/apiDefinition.swagger.json (hunk 1 above)
Comment (nikunjvf) [reply certified-connectors/Vineforce/apiDefinition.swagger.json]: Updated with latest commit

File: certified-connectors/Vineforce/apiDefinition.swagger.json (hunk 2 above)
Comment (nikunjvf) [reply certified-connectors/Vineforce/apiDefinition.swagger.json]: Updated with latest commit

File: certified-connectors/Vineforce/apiDefinition.swagger.json (hunk 3 above)
Comment (nikunjvf) [reply certified-connectors/Vineforce/apiDefinition.swagger.json]: Updated with latest commit

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**